contracts/StakeLockContract.json

```diff
@@ -0,0 +1,143 @@
+[
```

Comment:
I am wondering whether it makes sense to check-in the `json` file. We can always generate the `json` file from: `solc StakeLockContract.sol --abi`.

Reply:
We are checking in the generated files in general. But I will use the version from solc in next PR. I will remove this now.

Reply:
Sorry, are you planning to removing it now or in next PR?

Reply:
Offline @rlan35 re-iterated earlier consensus on keeping generated files. 
@harmony-ek points that `go get` breaks if there are no generated files. As `go get` skips `go generate` as it can run arbitrary command embedded in a potentially untrusted source repo on the target machine. e.g. we don't want to `go get` a repo having `//go:generate sudo rm -rf /`